.github/dependabot.yml

```diff
@@ -1,20 +1,20 @@
 version: 2
 updates:
-  - package-ecosystem: 'github-actions'
-    directory: '/'
+  - package-ecosystem: "github-actions"
+    directory: "/"
     schedule:
-      interval: 'monthly'
+      interval: "monthly"
     labels:
-      - 'skip-changelog'
```

Comment:
Just a note for when resolving conflicts: dependabot should not have the `skip-changelog` label